nwb_conversion_tools/datainterfaces/ecephys/neuralynx/neuralynxdatainterface.py

```diff
@@ -16,10 +77,22 @@ class NeuralynxRecordingInterface(BaseRecordingExtractorInterface):
     def __init__(self, folder_path: FolderPathType):
         self.subset_channels = None
         self.source_data = dict(folder_path=folder_path)
-        neuralynx_files = natsorted([str(x) for x in Path(folder_path).iterdir() if ".ncs" in x.suffixes])
-        extractors = [NeuralynxRecordingExtractor(filename=filename, seg_index=0) for filename in neuralynx_files]
+        nsc_files = natsorted([str(x) for x in Path(folder_path).iterdir() if ".ncs" in x.suffixes])
+        extractors = [NeuralynxRecordingExtractor(filename=filename, seg_index=0) for filename in nsc_files]
         gains = [extractor.get_channel_gains()[0] for extractor in extractors]
         for extractor in extractors:
             extractor.clear_channel_gains()
         self.recording_extractor = self.RX(extractors)
         self.recording_extractor.set_channel_gains(gains=gains)
+        try:
+            for i, filename in enumerate(nsc_files):
+                self.recording_extractor.set_channel_property(
+                    i,
+                    "filtering",
+                    get_filtering(filename),
+                )
+        except:
```

Comment:
What specific error occurs if `get_filtering` fails? Do you know?

It's a [E722](https://www.flake8rules.com/rules/E722.html) code style violation to use a bare except but as you know we can always use `BaseException` or `Exception` if we really need to (_i.e._, if we have no clue when/if it would fail)

Reply:
I'm just trying to account for a file that is older and doesn't have the right header form